llumnix/rpc/queue_client.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2024, Alibaba Group;
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from contextlib import contextmanager
+
+import zmq
+import zmq.asyncio
+import cloudpickle
+
+from llumnix.logger import init_logger
+from llumnix.server_info import ServerInfo
+
+from llumnix.rpc.utils import (RPC_GET_DATA_TIMEOUT_MS, RPC_SOCKET_LIMIT_CUTOFF, RPC_ZMQ_HWM, RPC_SUCCESS_STR,
```

Comment:
This variable is more like a config. Getting these from utils is weird.

Reply:
There some projects including constant variables in utils.py, including vllm. Currently, I don't want to add a seperate file for these few constants.